docs/user_guide/installation_and_version_guide/v0.7.1.rst

```diff
@@ -18,4 +18,6 @@ v0.7.1
 * :class:`~fairlearn.preprocessing.CorrelationRemover` now exposes
   :code:`n_features_in_` and :code:`feature_names_in_`.
 * Added the ACSIncome dataset and corresponding documentation.
-* Add `sphinxcontrib-bibtex` extension to manage citations in documentation using bibtex.
\ No newline at end of file
+* Add `sphinxcontrib-bibtex` extension to manage citations in documentation using bibtex.
```

Comment:
Heads up I'm renaming this to v0.8.0.rst in another PR and depending on who merges first this will create a conflict. Should be trivial to resolve, though.